src/Settings/Settings.php

```diff
@@ -102,4 +104,32 @@ public static function returnFormattedMetaboxFields( $postType ) {
 
 		return $result;
 	}
+	public static function returnColorCSS(){
```

Comment:
Kleinlich, ich weiß, aber bitte benenn die Funktion in getColorCSS um.

Reply:
Alles klar, war nur etwas verwirrt weil die Funktion darüber returnFormattedMetaboxFields heißt :)